Dockerfile

```diff
@@ -99,5 +86,12 @@ RUN chmod +x /entrypoint.sh
 # Let the user know how their container was built
 COPY dockerfiles/semgrep.Dockerfile /Dockerfile
 
+COPY --from=semgrep-core /semgrep/semgrep-core/_build/default/src/cli/Main.exe /usr/local/bin/semgrep-core
+
+ENV SEMGREP_IN_DOCKER=1 \
```

Comment:
why does this need to be set at the end instead of with all the other env vars?

Reply:
I do it so that we can decide to mess with these env vars and reuse all of the build cache from before